### ISSUE TITLE
Mixed content https with lets encrypt

### ISSUE THREAD

**Issue by ichasco** [containous/traefik#2267]
### Do you want to request a *feature* or report a *bug*?

BUG

### What did you do?

Put a nginx behind traefik

### What did you expect to see?

full https connection

### What did you see instead?

Https connection with mixed-content

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.3.8
Codename:     raclette
Go version:   go1.8.3
Built:        2017-09-07_08:46:19PM
OS/Arch:      linux/amd64

```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
accessLogsFile = "log/access.log"
logLevel = "ERROR"
checkNewVersion = true
debug = false

defaultEntryPoints = ["http", "https"]

[acme]
  email = "info@example.com"
  storageFile = "/etc/traefik/acme/acme.json"
  entryPoint = "https"
  acmeLogging = true
  onDemand = true
  OnHostRule = true
  
 [entryPoints]
  [entryPoints.http]
    address = ":80"
     [entryPoints.http.redirect]
       entryPoint = "https"
   [entryPoints.https]
     address = ":443"
       [entryPoints.https.tls]

[retry]
       
################################################################
# Web configuration backend
################################################################

[web]
  address = ":8080"
  
################################################################
# Docker configuration backend
################################################################

[docker]
  endpoint = "unix:///var/run/docker.sock"
  domain = "example.com"
  watch = true
  swarmmode = false

```

### If applicable, please paste the log output in debug mode (`--debug` switch)

```

```

**Comment by nmengin** [comment]
Hello @ichasco .

Can you give more information about your problem : 
* your bakend/frontend configuration 
* logs in DEBUG mode if possible.

Because your configuration SGTM.

**Note** : Both `onDemand` and `onHostRule` option are unnecessaries. If you used `Host` option into your frontends, you just have to use `onHostRule` option.

**Comment by w3blogfr** [comment]
I found a solution by upgrading traefik to 1.4.2 to use custom header.
Please see the example  https://github.com/w3blogfr/rancher-error-https-exemple

**Comment by sentriz** [comment]
@w3blogfr is your method still the best way of doing this?
can it be done with the new docker `traefik.headers.customrequestheaders` labels? if so, please let me know how. I would really appreciate it.
thank you

**Comment by w3blogfr** [comment]
I know they are homogenizing options between all providers #2785 
For docker provider, it's has been done by the ticket #2584 and merged in next 1.6

Whatever, I think custom label has already been added for docker provider in 1.5 (#2028)
So, it's should work now.

I'm sorry, I never tested 1.5 for that purpose, I'm waiting the 1.6 to migrate my application.

Regards.
